app/views/bookings/index.erb

```diff
@@ -19,11 +19,21 @@
                     <p><strong>Planet:</strong> <%= booking.planet.name %></p>
```

Comment:
the name of the planet should either be clickable or i would add a link to the planet